lib/jormungandr/test/data/jormungandr/config-for-passive-launch.yaml

```diff
@@ -0,0 +1,10 @@
+p2p:
+  trusted_peers: [
+        {
+          address: "/ip4/127.0.0.1/tcp/8081",
+          id: "1a563f5c367515f5eadc1c6c690d55320ef578fdd8c2bbe1"
+        }
+        ]
```

Comment:
Where's that trusted peer coming from though :no_mouth: ? 
Can this list be empty (not sure of Jörmungandr behavior in this scenario)? 

Also, I wonder about the potential value of such automated test. To be really interesting, we would have to connect to actual peers and see that we are processing blocks. Yet this requires a running network to connect to. I think that's something we'd better do as a nightly test, very much like the restoration test we used to have.

Reply:
> Where's that trusted peer coming from though no_mouth ?

From here -> https://github.com/input-output-hk/cardano-wallet/pull/956/files#diff-61049af2693330dca34f6fedf02ab74eR1-R5
(as the config.yaml is used to start the _main_ jormungandr in integration tests -> https://github.com/input-output-hk/cardano-wallet/pull/956/files#diff-6309c3e81be1c6f2739233773ccb247bR52-R54)

> Also, I wonder about the potential value of such automated test

The test launches a wallet and a node in a passive mode and connects to a main jormungandr. (Pretty much a Daedalus use case)
This check may be too weak -> https://github.com/input-output-hk/cardano-wallet/pull/956/files#diff-64ca464507dd45f6fad843c0767f1962R81-R85, yet, again, this is what Daedalus is doing. (see: https://github.com/input-output-hk/cardano-wallet/pull/1009#discussion_r345202730)

Reply:
I see. We can't actually rely on `8081` to always be available on the machine, especially in CI. 
This setup requires a bit more effort and is something we'll work out for the nightly tests! I'll open a corresponding ticket on the laboratory to capture this and close this PR if you don't mind ?

Reply:
Ok, would it make sense then to rework this PR and leave the first commit actually (without modifications to `config.yaml`)? It is about removing old test code.

Reply:
Hmmmm. But we still do use that code actually. The call to `genConfigYaml` is essential to make sure we run various executions of Jörmungandr using different state directory. 

The `public_address` part is probably obsolete now however, because this can now be removed entirely from the configuration file and we have no particular use for it.

Reply:
Wait a minute.. I think you're right actually :thinking: ... even the storage directory, it isn't taken from the configuration anymore...

Reply:
Correct. We are no longer generating `config.yaml` for jormungandr like we used to.

Reply:
Yes, so we can indeed keep that first commit modulo the `public_id` addition in the configuration file. 
Eventually, we should even get rid of:

```
            [ "--secret", testDataDir </> "secret.yaml"
            , "--config" , testDataDir </> "config.yaml"
            ]
```

which should be done at the call-site and not from this function.